polaris-service/src/test/java/org/apache/polaris/service/task/TableCleanupTaskHandlerTest.java

```diff
@@ -318,21 +345,35 @@ public void testTableCleanupMultipleSnapshots() throws IOException {
       Assertions.assertThatPredicate(handler::canHandleTask).accepts(task);
 
       CallContext.setCurrentContext(CallContext.of(realmContext, polarisCallContext));
+
       handler.handleTask(task);
 
-      assertThat(
-              metaStoreManagerFactory
-                  .getOrCreateMetaStoreManager(realmContext)
-                  .loadTasks(polarisCallContext, "test", 5)
-                  .getEntities())
+      List<PolarisBaseEntity> entities = metaStoreManagerFactory
```

Comment:
2. For the single metadata with multiple snapshot case, verify the ability to delete all of the statistics files